test/e2e/serial/tests/workload_placement.go

```diff
@@ -988,7 +988,9 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload placeme
 
 			By(fmt.Sprintf("checking the pods were scheduled with scheduler %q", corev1.DefaultSchedulerName))
 			for _, pod := range pods {
-				schedOK, err := nrosched.CheckPODWasScheduledWith(fxt.K8sClient, pod.Namespace, pod.Name, corev1.DefaultSchedulerName)
+				// starting v1beta1, if a pod is scheduled by the default scheduler, the ReportingController is left empty (unset)
+				// thus we pass an empty string as the expected scheduler that took charge of scheduling the workload.
+				schedOK, err := nrosched.CheckPODWasScheduledWith(fxt.K8sClient, pod.Namespace, pod.Name, "")
```

Comment:
I don't think `v1beta1` is relevant here. This is a behavior of the default scheduler, which is completely unaware of all the TAS machinery - so the latter is hardly relevant.
While I'm actually kinda OK with the fix proper, I'd like to have a look into the k8s codebase to learn a bit more, I find weird the default scheduler is not setting `ReportingController` in the event.

Reply:
I'll research more the root cause of not setting the def-sched and update here.